engine/src/main/resources/application.conf

```diff
@@ -78,86 +70,128 @@ call-caching {
   lookup-docker-hash = false
 }
 
-backend {
-  // Either "jes", "local", or "sge" (case insensitive)
-  defaultBackend = "local"
-  // List of backends which this Cromwell supports. Be sure to include the defaultBackend!
-  backendsAllowed = [ "local" ]
-
-  // If backend is JES, must supply a 'jes' stanza:
-  jes {
-    // project = ""
-    // baseExecutionBucket = ""
-    // endpointUrl = ""
-
-    // Polling for completion backs-off gradually for slower-running jobs.
-    // This is the maximum polling interval (in seconds):
-    // maximumPollingInterval = 600
-  }
-
-  // For any backend that assumes a local filesystem (local, sge)
-  shared-filesystem {
-    // Root directory where Cromwell writes job results to.  This directory must be
-    // visible and writeable to the Cromwell process as well as the jobs that Cromwell
-    // launches
-    root: "cromwell-executions"
-
-    // Cromwell makes a link to your input files within <root>/<workflow UUID>/workflow-inputs
-    // The following are strategies used to make those links.  They are ordered.  If one fails
-    // The next one is tried:
-    //
-    // hard-link: attempt to create a hard-link to the file
-    // copy: copy the file
-    // soft-link: create a symbolic link to the file
-    //
-    // NOTE: soft-link will be skipped for Docker jobs
-    localization: [
-      "hard-link", "soft-link", "copy"
-    ]
-  }
-
+system {
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
-  abortJobsOnTerminate = false
+  abort-jobs-on-terminate = false
 }
 
-// If authenticating with Google (e.g. if backend is 'JES') you must supply a top level stanza 'google':
-// This defines how all requests to JES/GCS performed by Cromwell will be authenticated.
 google {
-  applicationName = "cromwell"
-
-  // cromwellAuthenticationScheme can take the values "service_account",
-  // "user_account", and "application_default". The first two are described
-  // in the stanzas below.
-  // Use "application_default" to use the default service account credentials.
-  // This is useful if you are running on a GCE VM and if you don't need
-  // user level access. See https://developers.google.com/identity/protocols/application-default-credentials
-  // for more info. No further information is needed.
-  cromwellAuthenticationScheme = "application_default"
-
-  // If cromwellAuthenticationScheme is "service_account"
-  serviceAuth {
-    // pemFile = ""
-    // serviceAccountId = ""
-  }
 
-  // If cromwellAuthenticationScheme is "user_account"
-  userAuth {
-    // user = ""
-    // secretsFile = ""
-    // dataStoreDir = ""
-  }
+  application-name = "cromwell"
+
+  auths = [
+    {
+      name = "application-default"
```

Comment:
hrm.  I tried to be consistent in dashing the keys, the values seem to be kind of a mish mash.  Do you think everything should be dashes or just consistent?